src/scripts/install.py

```diff
@@ -143,7 +152,7 @@ class PercentSignTemplate(string.Template):
     else:
         app_exe = process_template('botan%{program_suffix}')
 
-    for d in [options.destdir, lib_dir, bin_dir, target_doc_dir, target_include_dir]:
+    for d in [options.prefix, lib_dir, bin_dir, target_doc_dir, target_include_dir]:
         makedirs(d)
```

Comment:
yes, makes sense. because the other dirs are prefixed with the absolute options.prefix instead of rel_prefix.